src/scroll-timeline-base.js

```diff
@@ -731,7 +733,13 @@ function parseInset(value) {
         throw TypeError(`Could not parse inset "${value}"`);
       }
     });
+  } else if (Array.isArray(value)) {
+    parts = value;
+  } else {
+    parts = [value];
   }
+
+
```

Comment:
nit: The double newline feels a bit odd here.

Reply:
Thanks. 🙏
Should be gone now.